.github/workflows/ecosystem-ci.yml

```
@@ -15,7 +15,7 @@ jobs:
       - uses: pnpm/action-setup@v4
       - uses: actions/setup-node@v4
         with:
-          node-version: 22
+          node-version: 'latest'
```

I would like to change this to `lts/*` when 22 becomes 22, so let's leave it here.
```suggestion
          node-version: 22
```